guides/src/main/java/io/micronaut/guides/core/asciidoc/AsciidocConfiguration.java

```diff
@@ -127,4 +127,11 @@ public interface AsciidocConfiguration {
      * @return the callouts directory
      */
     String getCalloutsDir();
+
+    /**
+     * Returns the base directory.
```

Comment:
```suggestion
     * Returns the base directory.
```

please, fix the javadoc